src/components/posts/PostCommentListItem.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import dateTimetoRelativeTime from './helperfunctions'
+import { Box, Stack } from '@mui/material'
+
+function CommentItem({ comment }) {
+  return (
+    <>
+      {comment.user && (
```

Comment:
Could you move this conditional rendering out of the main return block?

Something like: 
```js
if(!comment.user) {
   return <></>
}

return (
<>
  <li className='comment-item'>
    ...etc

```